--- a/Makefile
+++ b/Makefile
@@ -70,7 +70,7 @@
 	@echo 'Setting TEXTDOMAINDIR and TEXTDOMAIN variables'
 	sed -e "s'@MODIR@'$(MODIR_PLAIN)'g" -e "s'@EXENAME@'$(EXENAME)'g" \
 		-- '$<' >'$@'
-	chmod 755 -- '$@' || echo 'warn: chmod 755 $@ failed'
+	chmod 755 '$@' || echo 'warn: chmod 755 $@ failed'
 ifeq ($(MODIFY_SHEBANG), TRUE)
 	@echo 'Modifying shebang…'
 	sed -e "1s'"'^\#\!.*$$'"'$${SHEBANG:-#!`command -v sh 2>/dev/null`}'" \
